.github/workflows/roll_browser_into_playwright.yml

```diff
@@ -56,6 +70,7 @@ jobs:
         private-key: ${{ secrets.PLAYWRIGHT_PRIVATE_KEY }}
     - name: Create Pull Request
       uses: actions/github-script@v7
+      if: ${{ steps.check-branch.outputs.exists == '0' }}
```

Comment:
It seems like line 65 (new code) will still update the branch to an empty one, overwriting everything that previous bot did? Perhaps we should merge the new check in "Prepare branch" step instead?

Additionally, I think it's worth adding a `force` option to the workflow dispatch. Last time I tried to trigger something like this, I spent quite some time to figure out the branch name was clashing with an existing one, and the workflow failed to do anything.

Reply:
Moved it down to the "Prepare branch" step.

Removed the manual workflow dispatch event. Was added as a workaround (esrp release failure) which is not needed anymore.